pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3100,6 +3100,16 @@ type ClientSpec struct {
 	RemoveSecret bool `json:"removeSecret,omitempty"`
 	// +kubebuilder:pruning:PreserveUnknownFields
 	Caps map[string]string `json:"caps"`
+	// Security represents security settings
+	// +optional
+	Security ClientSecuritySpec `json:"security,omitempty"`
+}
+
+// ClinetSecuritySpec represents security settings for a Ceph Client
+type ClientSecuritySpec struct {
+	// CephX configures CephX key settings. More: https://docs.ceph.com/en/latest/dev/cephx/
```

Comment:
I'd suggest let's not link to the ceph docs unless it's directly helpful. No need to link general info docs.

Reply:
I think this is relevant because many users won't have any idea what CephX is. This line is copy-pasted from what I set up for RGW, and for consistency I think it makes sense to keep